worker/src/lib.rs

```diff
@@ -161,10 +163,21 @@ pub async fn witness_instagram_post(
     })
 }
 
+fn trim_newlines(s: &mut String) {
```

Comment:
There's a `trim` method for Strings. But I don't understand because the new lines are part of the signature target?

Reply:
Yes, the \n are signed in Twitter and Discord (but not in DNS or Instagram). For some reason, there is a difference in the new lines between the two implementations, so this is a kludge for short-term fix. Once the attestation generation is coming from one place, it'll either sort itself out or I'll solve it then. 

Trim wasn't working for some reason, that's what I tried first. Rather than debug the white space issue, my plan was to get a working fix, then clean it up